Sources/SentrySwiftUI/SentryTracedViewState.swift

```diff
@@ -0,0 +1,7 @@
+import Foundation
+
+class SentryTracedViewState {
```

Comment:
`m`: I don't see this file in Xcode. Did you forget to add it or maybe it can be deleted?

Reply:
Somehow I deleted it from the project but not into the trash. Thanks